src/main/java/com/dezorganizacja/bridge/view/graphical/Controller.java

```diff
@@ -0,0 +1,11 @@
+package com.dezorganizacja.bridge.view.graphical;
+
+import com.dezorganizacja.bridge.presenter.MainPresenter;
+
+public class Controller {
+    protected MainPresenter presenter;
```

Comment:
nie wiem, czy dobrze rozumiem ideę MainPresentera, rozumiem, że to ma być wszystkowiedząca (w szczególności wiedząca, kogo się pytać) klasa z którą powinny rozmawiać widoki? jeśli tak, to nie widzę konieczności, żeby tak nie robić